db/migrate/20160415121724_create_cloud_services.rb

```diff
@@ -0,0 +1,21 @@
+class CreateCloudServices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_services do |t|
+      t.string :ems_ref
+      t.string :source
+      t.string :binary
+      t.string :hostname
+      t.string :status
+      t.boolean :scheduling
```

Comment:
We have scheduling_disabled_reason there too... so this push me to using scheduling_disabled but I think scheduling_enabled seems more appropriate.... what do you think?